tool/template-validator-domain-verifier.html

```diff
@@ -0,0 +1,32 @@
+{% extends "template-base.html" %}
+
+{% block main %}
+<section class="container-fluid p-3">
+  <h1>Domain Verification Checker</h1>
+
+  <p>This tool allows you to verify that <a href="https://xrpl.org/xrp-ledger-toml.html#domain-verification">domain verification</a> is properly configured. </p>
```

Comment:
Use a relative, not absolute, link here.

Reply:
I embarrassingly keep making this mistake. Sorry!